CHANGELOG.md

```diff
@@ -13,5 +13,6 @@ release.
 
 - First draft of the OpenEdx E-Commerce service docker image (target release
   defaults to the `master` branch)
+- Job on circleci to push images on docker hub
```

Comment:
```suggestion
- Job on CircleCI to publish images on DockerHub
```

Reply:
s/images/image no ?

Reply:
Nope, because there is also the `latest` image for tagged releases.

Reply:
> ok boomer